prdoc/pr_2226.prdoc

```diff
@@ -14,7 +15,14 @@ doc:
       will be used on Kusama and Polkadot. In nutshell `UpToLimitDisablingStrategy`
       disables offenders up to the configured threshold. Offending validators are not disabled for
       offences in previous eras. The threshold is controlled via `DISABLING_LIMIT_FACTOR` (a generic
-      parameter of `UpToLimitDisablingStrategy`).
+      parameter of `UpToLimitDisablingStrategy`). Also offending validators are no longer chilled
+      after an offence is committed because a malicious validator can unchill itself and continue
+      committing offences.
```

Comment:
```suggestion
      parameter of `UpToLimitDisablingStrategy`). Also offending validators are no longer chilled
      after an offence is committed. This change is made to protect the network in case of honest nodes being slashed.
```